python/paddle/fluid/__init__.py

```diff
@@ -21,8 +21,7 @@
 from executor import *
 
 import trainer
-from trainer import Trainer
-from trainer import Event
+from trainer import *
```

Comment:
No `import *` please, due to https://shahriar.svbtle.com/importing-star-in-python

Reply:
Fixed in https://github.com/PaddlePaddle/Paddle/pull/10354